File: Sample/.Net/PublicApiDemo/Common/PlexePublicAPIDemo.Common/AuthenticationService.cs
using Newtonsoft.Json;
using PlexePublicAPIDemo.Common.Model;
using RestSharp;
using System;
using System.Collections.Generic;
using System.Net;
using System.Text;

namespace PlexePublicAPIDemo.Common
{
    public class AuthenticationService
    {
        private readonly string BASE_URI = "https://apidemo.plexe.co/";
        private readonly string APPLICATION_URI = "api/Authenication/";
        public UserData AuthenticateUser(string username, string password)
        {
            UserData userDatas = new UserData();
            userDatas = null;
            try
            {
                var client = new RestClient(BASE_URI + APPLICATION_URI + "login?username=" + username + "&password=" + password + "");
                var request = new RestRequest();
                request.Method = Method.Get;
                request.AddHeader("Accept", "application/json");
                RestResponse response = client.Execute(request);
                userDatas = JsonConvert.DeserializeObject<UserData>(response.Content);
                return userDatas;
            }
            catch (Exception ex)
            {
                return null;
            }
        }
    }    
}

File: Sample/Php/PublicApiDemo/services/AuthenticationService.php
<?php

require_once 'HTTP/Request2.php';
require_once 'vendor/autoload.php';
  
use GuzzleHttp\Client;
use GuzzleHttp\Exception\RequestException;
use GuzzleHttp\Psr7\Request;
  
class AuthenticationService
{
    private $BASE_URI = "https://apidemo.plexe.co/";
    private $APPLICATION_URI ="api/Authenication/";

    function StatusCodeHandling($e)
    {
        if ($e -> getResponse() -> getStatusCode() == '400')
        {
            return $response;
            //return 'BAD REQUEST';
            //$this->prepare_access_token();
        }
        else if ($e -> getResponse() -> getStatusCode() == '422')
        {
            $response = json_decode($e -> getResponse()->getBody(true)->getContents());
            return $response;
        }
        else if ($e -> getResponse() -> getStatusCode() == '500')
        {
            $response = json_decode($e ->getResponse()->getBody(true)->getContents());
            return $response;
        }
        else if ($e -> getResponse() -> getStatusCode() == '401')
        {
            $response = json_decode($e ->getResponse()->getBody(true)->getContents());
            return $response;
        }
        else if ($e->getResponse()->getStatusCode() == '403')
        {
            $response = json_decode($e->getResponse()->getBody(true)->getContents());
            return $response;
        }
        else
        {
            $response = json_decode($e->getResponse()->getBody(true)->getContents());
            return $response;
        }       
    }
    public function AuthenticateUser($username,$password){
        try
        {
            $client = new Client();
            $headers = [
                'Accept' => 'application/json'
            ];
            $request = new Request('GET', $this -> BASE_URI.$this -> APPLICATION_URI.'login?username='.$username.'&password='. $password, $headers);
            $res = $client->sendAsync($request)->wait();
            $result = $res->getBody();
            $decoded = json_decode($result);
            return $decoded;                
        }
        catch (RequestException $e)
        {
            $response =  $this->StatusCodeHandling($e);
            return $response;
        }
    }      
}
?>